### PR TITLE
exporters/ocagent: Add zPages to the example.

### PR DESCRIPTION
**Pull request by songy23**

### CODE REVIEW

File: content/exporters/supported-exporters/Go/ocagent.md [hunk 1]
```diff
@@ -170,6 +170,14 @@ import (
 )
 
 func main() {
+	zPagesMux := http.NewServeMux()
+	zpages.Handle(zPagesMux, "/debug")
```
Comment (odeke-em) [inline content/exporters/supported-exporters/Go/ocagent.md]: We forgot to import zpages in the import block.

Comment (songy23) [reply content/exporters/supported-exporters/Go/ocagent.md]: Good catch, fixed.

**Comment by songy23** [comment]
> is this more of a diagnostic for users?

Yes, this will be helpful if users ran the ocagent exporter but couldn't find spans/metrics on Agent.

**Comment by songy23** [comment]
> In that case, let's explicitly comment this code and state that zPages are only here
for debugging and link them for how to use zPages. As is they'll wonder what to do with zPages.

Make sense to me, added to comments.

File: content/exporters/supported-exporters/Go/ocagent.md [hunk 2]
```diff
@@ -172,8 +172,8 @@ import (
 
 func main() {
 	// For the purposes of debugging, we'll add zPages that you can
-  // use as a diagnostic to examine if stats and traces are exported
```
Comment (odeke-em) [inline content/exporters/supported-exporters/Go/ocagent.md]: Go fmt should work instead of manually replacing these :)

**Pull request merged**